CHANGELOG.asciidoc

```diff
@@ -123,6 +123,7 @@ https://github.com/elastic/beats/compare/v6.4.0...master[Check the HEAD diff]
 - Add tag "multiline" to "log.flags" if event consists of multiple lines. {pull}7997[7997]
 - Add haproxy module. {pull}8014[8014]
 - Release `docker` input as GA. {pull}8328[8328]
+- Rename user_agent.raw to user_ageint.original to follow ECS conventions. {pull}8537[8537]
```

Comment:
If this hasn't been released yet, what about editing the previous entry instead of adding two for the same thing?

Reply:
++